2021q3/clusteradm.adoc

```diff
@@ -0,0 +1,32 @@
+=== Cluster Administration Team
+
+Contact: Cluster Administration Team <clusteradm@FreeBSD.org>
+
+Links: +
+link:https://www.freebsd.org/administration/#t-clusteradm[Cluster Administration Team members] URL: link:https://www.freebsd.org/administration/#t-clusteradm[https://www.freebsd.org/administration/#t-clusteradm]
+
+The FreeBSD Cluster Administration Team consists of the people responsible for administering the machines that the Project relies on for its distributed work and communications to be synchronised. In this quarter, the team has worked on the following:
```

Comment:
I'd rewrite the first sentence as "The FreeBSD Cluster Administration Team is responsible for managing the machines the Project relies on to synchronise its distributed work and communications." Or perhaps "FreeBSD Cluster Administration Team members are responsible..." (but check I didn't misunderstand what you mean.)